stdlib/_curses.pyi

```diff
@@ -460,9 +460,9 @@ class _CursesWindow:
     def idlok(self, yes: bool) -> None: ...
     def immedok(self, flag: bool) -> None: ...
     @overload
-    def inch(self) -> _chtype: ...
+    def inch(self) -> bytes: ...
     @overload
-    def inch(self, y: int, x: int) -> _chtype: ...
+    def inch(self, y: int, x: int) -> bytes: ...
```

Comment:
`inch()` returns int (in C it returns `unsigned long`, which gets converted to Python `int`): https://github.com/python/cpython/blob/46c8d915715aa2bd4d697482aa051fe974d440e1/Modules/_cursesmodule.c#L1708